packages/backend/src/core/reports/createReport.ts

```diff
@@ -29,9 +30,10 @@ export function createReport(
     timestamp: price.timestamp,
     projectId: balance.projectId,
     asset: balance.assetId,
-    balance: balance.balance,
-    balanceUsd,
-    balanceEth,
+    type: balance.type,
+    amount: balance.balance,
+    usdValue: balanceUsd,
+    ethValue: balanceEth,
```

Comment:
I know that `convertBalance` returns `{ balanceUsd, balanceEth }` but I think it can be changed so that it returns `{ usdValue, ethValue }`. Also if you do so, maybe rename the function to be something like `convertValue`?